app/components/billing-status-display/billing-status-badge.hbs

```diff
@@ -0,0 +1,25 @@
+<div ...attributes>
+  {{#if this.shouldShowVipBadge}}
+    <Header::VipBadge @size={{@size}} />
+  {{else if this.shouldShowMemberBadge}}
+    <Header::MemberBadge @size={{@size}} />
+  {{else if this.activeDiscountForYearlyPlan}}
+    {{#if this.canSeeDiscountCountdown}}
+      <BillingStatusDisplay::DiscountTimerBadge @discount={{this.activeDiscountForYearlyPlan}} @size={{@size}} />
+    {{/if}}
+  {{else if this.shouldShowFreeWeeksLeftButton}}
+    <Header::FreeWeeksLeftButton @size={{@size}} />
+  {{else if this.shouldShowUpgradeButton}}
+    <PrimaryLinkButton
+      @size={{this.linkButtonSize}}
+      @route="pay"
+      class="{{if (eq this.linkButtonSize 'extra-small') 'mr-3'}}"
```

Comment:
@ryan-gang why do we have to specify `mr-3` manually here? Shouldn't the `...attributes` in the parent cover this?